test/jdk/javax/swing/JTable/PrintManualTest_FitWidthMultiple.java

```diff
@@ -114,38 +116,44 @@ public void run() {
                 instructFrame.setBounds(600,100,350,350);
 
                 /* Print Button */
-                final JButton printButton=new JButton("Print");
+                final JButton printButton = new JButton("Print");
 
                 /* Table Model */
-                final TableModel datamodel=new AbstractTableModel(){
+                final TableModel datamodel = new AbstractTableModel(){
                     @Override
-                    public int getColumnCount() { return 50;}
+                    public int getColumnCount() {
+                        return 50;
+                    }
                     @Override
-                    public int getRowCount() { return 50; }
+                    public int getRowCount() {
+                        return 50;
+                    }
                     @Override
-                    public Object getValueAt(int row, int column){ return new Integer(row*column);}
+                    public Object getValueAt(int row, int column) {
+                        return Integer.valueOf(row*column);
+                    }
                 };
 
                 /* Constructing the JTable */
-                final JTable table=new JTable(datamodel);
+                final JTable table = new JTable(datamodel);
 
                 /* Putting the JTable in ScrollPane and Frame Container */
-                JScrollPane scrollpane=new JScrollPane(table);
+                JScrollPane scrollpane = new JScrollPane(table);
                 fr = new JFrame("PrintManualTest_FitWidthMultiple");
                 fr.getContentPane().add(scrollpane);
 
                 /* Light Weight Panel for holding Print and other buttons */
-                JPanel btnpanel=new JPanel();
+                JPanel btnpanel = new JPanel();
                 btnpanel.add(printButton);
                 fr.getContentPane().add(btnpanel,BorderLayout.SOUTH);
```

Comment:
```suggestion
                fr.getContentPane().add(btnpanel, BorderLayout.SOUTH);
```
May be you can fix this here and other places as well. space after `,`.